src/module.rs

```diff
@@ -4,6 +4,25 @@ use ansi_term::Style;
 use ansi_term::{ANSIString, ANSIStrings};
 use std::fmt;
 
+// List of all modules
+pub const ALL_MODULES: &[&str] = &[
+    "battery",
+    "character",
+    "cmd_duration",
+    "directory",
+    "git_branch",
+    "git_status",
+    "golang",
+    "jobs",
+    "line_break",
+    "nodejs",
+    "package",
+    "python",
+    "ruby",
+    "rust",
+    "username",
```

Comment:
Would it make sense to get this dynamically instead?

Reply:
As it is, there is no way to get this dynamically.
It may be a good idea to make this an enum in a future refactor. 👍